decompiler/structures/pseudo/expressions.py

```diff
@@ -250,6 +250,20 @@ def copy(self) -> ExternFunctionPointer:
         return ExternFunctionPointer(self.value, self._type.copy(), self.tags)
 
 
+class NotUseableConstant(Constant):
+    """Represents a non useable constant like 'inf' or 'NaN' as a string"""
+    def __init__(self, value: str, tags: Optional[Tuple[Tag, ...]] = None):
+        super().__init__(value, UnknownType(), tags=tags)
```

Comment:
I am also not sure about the type. Technically it is a float or? We could also choose a custom type here. Choosing `UnkownType` as default seems wrong.

Reply:
Yes they are all floats in python. 
Would choose a custom type. 
Any name/size suggestions?

Reply:
I would say either float, since they are of type float, or some custom type, but there I am not sure about the name. Something like "special constant" seems to be a strange name.

Reply:
How about `CustomType("double")`? 
Technically correct, but will not be used as a real double :-).